src/geovista/bridge.py

```diff
@@ -708,6 +735,127 @@ def from_points(
+            if n_vecs not in (2, 3):
+                msg = (
+                    "Keyword 'vectors' must be a tuple of 2 or 3 arrays : "
+                    f"got {n_vecs}."
+                )
+                raise ValueError(msg)
+
+            xx, yy = vectors[:2]
+            zz = vectors[2] if n_vecs > 2 else np.zeros_like(xx)
+
+            if vectors_crs is None:
+                # Vectors crs defaults to main input CRS
+                # NOTE: vectors may have a different CRS than the input points.
+                # The only likely usage is for true-lat-lon winds with locations on a
+                # rotated grid or similar, but we probably do need to support that.
+                vectors_crs = crs
+
+            if vectors_crs is not None:
+                vectors_crs = pyproj.CRS.from_user_input(vectors_crs)
+                if vectors_crs == WGS84:
+                    # As for main crs, "None" means plain-latlon crs
+                    vectors_crs = None
+
+            if vectors_crs is None:
+                vector_xs = xs_latlon
+                vector_ys = ys_latlon
+                post_rotate_matrix = None
+            else:
+                # Supplied vector xx/yy/zz are for a "different" orientation than
+                # standard lat-lon, so will need rotating afterwards.
+                # NOTE: we can only do this for specific CRS types where we know how to
+                # find **its** pole.
+                axis_names = [x.name for x in vectors_crs.axis_info]
+                if axis_names[:2] != ["Longitude", "Latitude"]:
+                    msg = (
+                        "Cannot determine wind directions : "
+                        f"Target CRS is not supported : {vectors_crs}."
+                    )
+                    raise ValueError(msg)
+
+                # For a CRS with longitude and latitude axes, we treat its coordinates
+                # as latitudes and longitudes and (0, 90) as its North Pole.
+                # This means we can calculate the xyz vectors for the input points in
+                # vector coordinates, and afterwards rotate them to the display.
+                # TODO @pp-mo: identify add support for other common cases as needed,
+                #  for example OSGB.  Sadly I think there is no general solution.
+
+                # Find the true-latlon position of this CRS' North pole point.
+                transformed = transform_points(
+                    src_crs=vectors_crs, tgt_crs=WGS84, xs=0.0, ys=90.0
+                )
+                (
+                    pole_lon,
+                    pole_lat,
+                ) = transformed[:, 0][0], transformed[:, 1][0]
+                # We use this to program a "post-rotation" of the vectors.
+                from scipy.spatial.transform import Rotation
+
+                # NOTE: the axes orientation is odd -- see "vectors_to_cartesian"
+                rotmat_to_standard_pole = Rotation.from_euler(
```

Comment:
Why is this only applied when `vector_crs` is not None? surely you would expect to have the same issues with `vectors_to_cartesian` in both cases.

Reply:
I believe the correct matrix can be made with the following lines of code:
```python
bases_x = np.array([0, 90, 0])
bases_y = np.array([0, 0, 90])
transformed_bases = transform_points(src_crs=rp, tgt_crs=pc, xs=bases_x, ys=bases_y)
cartesian_bases = to_cartesian(tranformed_bases[:, 0], transformed_bases[:, 1])
post_rotate_matrix = np.matrix(cartesian_bases).T
```

Reply:
> I believe the correct matrix can be made with the following lines of code ....

Proposed : see https://github.com/pp-mo/geovista/pull/1